README.md

```diff
@@ -1,23 +1,52 @@
-# Hanami
+[![Gem Version](https://badge.fury.io/rb/hanami.svg)](http://badge.fury.io/rb/hanami)
+[![Build Status](https://secure.travis-ci.org/hanami/hanami.svg?branch=master)](http://travis-ci.org/hanami/hanami?branch=master)
+[![Coverage](https://coveralls.io/repos/hanami/hanami/badge.svg?branch=master)](https://coveralls.io/r/hanami/hanami)
+[![Code Climate](https://codeclimate.com/github/hanami/hanami.svg)](https://codeclimate.com/github/hanami/hanami)
+[![Dependencies](https://gemnasium.com/hanami/hanami.svg)](https://gemnasium.com/hanami/hanami)
+[![Inline docs](http://inch-ci.org/github/hanami/hanami.svg)](http://inch-ci.org/github/hanami/hanami)
+
+# Hanami :cherry_blossom:
 
 The web, with simplicity.
 
 ## Frameworks
```

Comment:
I think @jodosha refers to the individual parts as (single-purpose) frameworks. I typically think of a framework as being more general purpose, like `hanami/hanami`, and consider the other parts "libraries". What do you think, Luca?